src/main/java/cz/cvut/kbss/termit/model/comment/Comment.java

```diff
@@ -0,0 +1,151 @@
+package cz.cvut.kbss.termit.model.comment;
+
+import cz.cvut.kbss.jopa.model.annotations.*;
+import cz.cvut.kbss.termit.exception.TermItException;
+import cz.cvut.kbss.termit.model.AbstractEntity;
+import cz.cvut.kbss.termit.model.User;
+import cz.cvut.kbss.termit.util.Vocabulary;
+
+import java.lang.reflect.Field;
+import java.net.URI;
+import java.util.Date;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * Represents a single comment in the comment/discussion module.
+ */
+@OWLClass(iri = "http://rdfs.org/sioc/types#Comment")
```

Comment:
Constants would be better.

Reply:
Where should I put such constants?

Reply:
Simply to Constants.java? You can create a custom class of course, or even declare them in the termit ontology.

Reply:
I'm considering adding import of SIOC and ActivityStreams into the TermIt glossary/vocabulary. That would solve it. What do you think?

Reply:
Best option IMO.